static/open-specs/liveobjects.yaml

```diff
@@ -0,0 +1,1386 @@
+          schema:
+            type: boolean
+            default: false
+        - name: limit
+          in: query
+          required: false
+          description: Set the number of objects to be returned in each page.
+          schema:
+            type: integer
+        - name: cursor
+          in: query
+          required: false
+          description: The cursor used for pagination.
+          schema:
+            type: string
+        - name: metadata
+          in: query
+          required: false
+          description: Include object metadata in the response.
+          schema:
+            type: boolean
+            default: false
+      responses:
+        '200':
+          description: List of retrieved objects or object references.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: array
+                    items:
+                      $ref: '#/components/schemas/ObjectWithDetails'
+                  - type: array
+                    items:
+                      type: string
+                      example: "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
+              examples:
+                objectArray:
+                  summary: Full response with object values (url param values=true)
+                  value:
+                    - objectId: "map:YIffJYRAP2k2e7ZP+xzequ9c5kDu1LfI/sEOKoWHvv4@1742479683000"
+                      map:
+                        entries:
+                          myMapKey: { data: { string: "my map value" }}
+                          myBoolean: { data: { boolean: true }}
+                          myEncodedKey: { data: { bytes: "TGl2ZU9iamVjdHMgaXMgYXdlc29tZQo=", encoding: "base64" }}
+                    - objectId: "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
+                      map:
+                        entries:
+                          counter: { data:{ number: 5 }}
+                          myObjectRef: { data: { objectId: "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000" }}
+                refArray:
+                  summary: Just object references (url param values=false or not set)
+                  value:
+                    - "root"
+                    - "map:YIffJYRAP2k2e7ZP+xzequ9c5kDu1LfI/sEOKoWHvv4@1742479683000"
+                    - "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/AuthError'
+                  - $ref: '#/components/schemas/ClientRestrictionNotSatisfied'
+                  - $ref: '#/components/schemas/InvalidTokenKey'
+                  - $ref: '#/components/schemas/MalformedCredential'
+                  - $ref: '#/components/schemas/OperationObjectSubscribeUnauthorized'
```

Comment:
some of these errors are not rendering: 
<img width="666" alt="image" src="https://github.com/user-attachments/assets/8890621b-9505-4c10-8e78-8f6892cb4d84" />
<img width="675" alt="image" src="https://github.com/user-attachments/assets/36062e2e-8690-4e21-8896-7f27a9f20ab4" />

.. etc